### DISCUSSION TITLE
2023/02/19/clear-comments/

### DISCUSSION THREAD

**Discussion by giscus** [argvchs/giscus-comments#1] [Announcements]
# 2023/02/19/clear-comments/

https://argvchs.netlify.app/2023/02/19/clear-comments/

**Comment by wzdc** [comment]
之前在 [Hexo 博客使用 LaTeX](https://argvchs.netlify.app/2022/06/17/hexo-latex/) 的文章里评论，上传的图片中包含的路径没有打码，你帮我打码了。

（好奇）为什么博客路径都要打码呢？
